requirements.txt

```
@@ -14,3 +14,5 @@ soupsieve==2.2.1
 typing-extensions==3.7.4.3
 urllib3==1.26.5
 yarl==1.6.3
+dislash.py==1.0.17
+discord.py[voice]==1.3.7
```

```suggestion
discord.py[voice]==1.7.3
pynacl==1.4.0
```
Außerdem sollte Zeile 7 `discord.py==1.7.1` dann entfernt werden.
`pynacl` wird bei mir für den Zugriff auf ffmpeg benötigt (MacOS) da weiß ich nicht wie es auf Linux aussieht.